autogen/agentchat/groupchat.py

```diff
@@ -51,6 +51,11 @@ def custom_speaker_selection_func(
                 last_speaker: Agent, groupchat: GroupChat
             ) -> Union[Agent, str, None]:
             ```
+    - requery_on_multiple_speaker_names: whether to requery the LLM if multiple speaker names are returned during speaker selection.
+        The response with multiple names will be fed back to the LLM with a prompt asking it to select the primary speaker's name.
+        Applies only to "auto" speaker selection method.
+        Default is False, in which case if the LLM returns multiple speaker names it will not requery the LLM.
```

Comment:
What about making the default True? It seems to be a desirable choice and most users don't need to change the default.

Reply:
That sounds good to me, the more things that work out of the box with defaults, the better. I'll change it.